charts/forseti-security/values.yaml

```diff
@@ -18,41 +18,58 @@ cloudsqlConnection: ""
 # Whether or not to deploy a load balancer: none, external, internal
 loadBalancer: none
 
+# Whether or not to deploy the orchestrator.
```

Comment:
Nit- doc strings should start with the name of the property it describes for better clarification: https://helm.sh/docs/chart_best_practices/#document-values-yaml